### ISSUE TITLE
GLTFNode often gives wrong transformation

### ISSUE THREAD

**Issue by kusma** [cables-gl/cables_docs#393]
**Describe the bug**
Sometimes GLTFNodes don't appear in the right location.

**To Reproduce**
1. Open [this patch](https://cables.gl/p/636032632dde22033587e08b?s=popk5xE2ncgSJP8H)
2. Observe how the red bunny and the blue normals don't line up

**Expected behavior**
The red bunny should be located where the normals are shown.

**Screenshots**
![image](https://user-images.githubusercontent.com/47073/199107083-9bd18e31-a67a-4df4-8acb-6b7dc82a8dbd.png)

**Additional context**
Previously reported on Discord

**Comment by pandrr** [comment]
its not an absolute transform, you have to click "hierarchy" to create nodes for the complete transform chain.

gltfnode is basically only for the mesh itself

**Comment by kusma** [comment]
Then why does it have a "Transformation"-checkbox, and what makes it different from GLTFGeometry, then?

**Comment by kusma** [comment]
If it's only for the mesh, then it's poorly named IMO. "Node" clearly implies a transformation + geometry pair.

**Comment by kusma** [comment]
Looking at the code, it does indeed seem to try to apply the transform. I guess you're just confusing it with GltfGeometry...

**Comment by AMajesticSeaFlapFlap** [comment]
> If it's only for the mesh, then it's poorly named IMO. "Node" clearly implies a transformation + geometry pair.

Node in gltf specification only refers to `An object defining the hierarchy relations and the local transform of its content.` https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html

**Comment by pandrr** [comment]
clicking hierarchy it gives the correct result.
yes there is no parent element, as i thought, it should be fine with one gltfnode op

**Comment by kusma** [comment]
> clicking hierarchy it gives the correct result.

I assume you mean to *uncheck* it. 

But no, it does not. It just makes both objects incorrectly positioned. You can check "Draw" on the GLTFScene object to see where the object should be.

**Comment by kusma** [comment]
> Node in gltf specification only refers to An object defining the hierarchy relations and the local transform of its content.

Yes exactly. That "and the local transformation" bit is what I mean. This is *not* just the mesh.

**Comment by kusma** [comment]
Looking at the code, and forking the node, it seems like a pretty simple bug: the second argument to node.draw is `dontTransform`,  not "doTransform" like the code seems to assume.

So the negation of `inTrans.get()` is incorrect here.
